extensions/wikia/AuthPreferences/AuthPreferencesModuleService.php

```diff
@@ -30,6 +30,8 @@ public function renderAuthPreferences() {
 		$googleConnectDomain = wfGetBaseDomainForHost( $wgServer );
 		$googleConnectUrl = WikiFactory::getLocalEnvURL( "https://www.$googleConnectDomain/google-connect" );
 		$this->setVal( 'googleConnectAuthUrl', $googleConnectUrl );
+		$twitchConnectUrl = WikiFactory::getLocalEnvURL( "https://www.$googleConnectDomain/twitch-connect" );
```

Comment:
I would not use `googleConnectDomain` for both Google and Twitch - it may be confusing and lead to problems when someone changes this url wanting to affect only Google login.